src/styles/main.scss

```diff
@@ -0,0 +1,57 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  margin: auto;
+  width: 600px;
+  height: 600px;
+  position: relative;
+  border: 2px solid black;
+  background-color: aquamarine;
+  transform: rotate(180deg);
+}
+
+@keyframes rotate {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+#container-min {
+  width: 10px;
+  height: 180px;
+  background-color: brown;
+  border-radius: 5px;
+  animation: rotate 3600s linear infinite;
+}
+
+#container-sec {
+  width: 10px;
```

Comment:
create mixins and variables for repetitive styles

Reply:
done